pages/volunteer/volunteer.html

```diff
@@ -0,0 +1,479 @@
+<!DOCTYPE html>
+<section id="volunteer" ng-controller="volunteerPage">
+  <md-card flex layout-gt-sm="row" layout="column" layout-fill class="oppia-foundation-green-background oppia-foundation-vertical-padding volunteer-page-conditional-horizontal-padding">
+    <md-card-content layout="column" flex="50">
+      <md-content layout="column" class ="oppia-foundation-white-text oppia-foundation-green-background">
+        <md-card-title>
+          <md-card-title-text>
+            <h1 class="md-display-3" hide-xs>
+              Join our global team of volunteers
+            </h1>
+            <h3 class="md-display-1" show-xs hide-gt-xs>
+              Join our global team of volunteers
+            </h3>
+          </md-card-title-text>
+        </md-card-title>
+        <h4 class="md-headline volunteer-page-title-box" hide-xs>
+          Contributing to Oppia is an opportunity to impact the lives of millions of students.
+        </h4>
+        <h4 class="md-headline volunteer-page-title-box" hide-xs>
+          It doesn’t matter where you come from, what language you speak, or how young or old you are - this is a community project, and all are welcome!
```

Comment:
Should we change the "docs" quote marks `’` to `'`, here and elsewhere?

I am thinking that if we were to edit the page in the future, we would use `'`, and that would cause an inconsistency if we left the fancy quote marks in.

Reply:
IMO, I don't think we should. Here is the before:
![screen shot 2018-07-31 at 7 41 30 am](https://user-images.githubusercontent.com/11153258/43458098-f106057a-9496-11e8-9e0a-3402b96cefbc.png)
and after:
![screen shot 2018-07-31 at 7 44 09 am](https://user-images.githubusercontent.com/11153258/43458105-f5194334-9496-11e8-9958-e5631f476769.png)

To me, the `before` photo looks easier on the eyes.

I have been using Google Docs/ MS Word to type these lengthy chunks of text to help prevent typos and grammar mistakes. That is where these fancy quote marks come from. 
That is just me though. Other dev might just type it out using their text editors/ IDEs and uses the regular `'`.

Typography is important for the English language. This is not code styling. 
Anyway, thats my lengthy /rant. Lmk what you think @seanlip, you are the final voice here :)

Reply:
As a compromise,I will use`&lsquo;` and `&rsquo;`.

Reply:
I don't have strong feelings about this. Happy for you to decide!